tagged_visitor/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "tagged_visitor"
+version = "0.1.0"
+authors = ["Mr-Andersen <andrew-anderson.neo@yandex.ru>"]
```

Comment:
Possibly we should both be here, as in https://github.com/teloxide/mixed-tags-serde/blob/dev/Cargo.toml#L4?

Also, what's the point of splitting this crate up into 2 crates?

Reply:
This is weird. I have only your name in that line. And also `name = "mixed-tag-serde"`

This splitting is temporary. Thing is, the `proc-macro` crate can't export anything more except one macro. We could dump whole `TaggedVisitor` definition in-place, but this would lead to repeating definition (probably could be fixed) and just more code (harder to debug?). I used additional crate to be able to `use` it in test.